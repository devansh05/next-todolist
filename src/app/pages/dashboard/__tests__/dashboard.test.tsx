import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import "@testing-library/jest-dom";
import Dashboard from '../dashboard';

describe('Dashboard', () => {
  it('renders dashboard', () => {
    // render(<Dashboard />)
    // const heading = screen.getByRole('heading', { level: 1 })
    // const heading = screen.getByTestId('heading')
    // const heading = screen.getByText('Home')
    // expect(heading).toBeInTheDocument()
    
    //Snapshot testing
    const tree = renderer.create(<Dashboard />).toJSON();
    expect(tree).toMatchSnapshot();
  })
})