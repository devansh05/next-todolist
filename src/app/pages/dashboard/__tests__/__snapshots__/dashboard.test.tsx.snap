// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard renders dashboard 1`] = `
<h1
  data-testid="heading"
>
  Home
</h1>
`;
